### ISSUE TITLE
Please allow restricting to a certain directory and/or excluding directories

### ISSUE THREAD

**Issue by akorn** [alexdantas/kure-music-player#2]
Hi,

while kmp works great, unfortunately it finds various kinds of warning tones saved in other applications' directories and plays those as well in 'shuffle all songs' mode.

I'd like to either only play stuff from a specific directory, or exclude the directories of these other applications.

**Comment by lgommans** [comment]
Great idea, I was looking for this too!

I've done this in my fork, see this commit: https://github.com/lgommans/kure-music-player/commit/4d6be1363cf5a9b3e904054956138401025164f8

I haven't added a GUI option though. If you are still interested in this, I could see if I can get that to work. For now, to enable/change the setting, you'll have to dig into your `/data/data/com.kure.musicplayer/shared_settings` folder. I can also drop you the `.apk` file so you don't have to compile it yourself.

**Comment by akorn** [comment]
Thanks; I'm glad someone took the time to implement this.

I don't use the program anymore, though.

**Comment by GReagle** [comment]
Yes I would love this feature.  I just started using Kure, which I got from F-Droid.  I use the Random Shuffle feature and I do not want to hear audio books when listening to music.  I want to restrict to just the Music folder.  It is okay with me not to have a GUI.  I can edit text files.

Yes please I would like an .apk so I can install this.
